inc/namespace.php

```diff
@@ -19,15 +20,52 @@ function bootstrap() {
 function on_plugins_loaded() {
 	$config = get_config()['modules']['dev-tools'];
 
-	if ( $config['query-monitor'] ) {
-		// In Cloud environments, disable the Query Monitor error handler
-		// as this will override our own error handler in Cloud.
-		if ( in_array( get_environment_architecture(), [ 'ec2', 'ecs' ], true ) ) {
-			define( 'QM_DISABLE_ERROR_HANDLER', true );
-		}
+	if ( ! $config['query-monitor'] ) {
+		return;
+	}
+
+	// In Cloud environments, disable the Query Monitor error handler
+	// as this will override our own error handler in Cloud.
+	if ( in_array( get_environment_architecture(), [ 'ec2', 'ecs' ], true ) ) {
+		define( 'QM_DISABLE_ERROR_HANDLER', true );
+	}
+
+	// Hide the db.php dropin installation warning and prompt.
+	add_filter( 'qm/show_extended_query_prompt', '__return_false' );
+	require_once ROOT_DIR . '/vendor/johnbillion/query-monitor/query-monitor.php';
+
+	// Add Altis config info to the Query Monitor.
+	// Register collector.
+	add_filter( 'qm/collectors', __NAMESPACE__ . '\\register_qm_collector_altis_config' );
+
+	// Register outputter.
+	add_filter( 'qm/outputter/html', __NAMESPACE__ . '\\register_qm_output_html_altis_config', 120, 2 );
```

Comment:
Regarding the priority setting this doesn't necessarily need to be all the way at the bottom of the menu. Might be good to surface it to just after the "Overview" tab IMO.